### ISSUE TITLE
Get SPE file location always from importlib resources absolute path

### ISSUE THREAD

**Issue by morcuended** [cta-observatory/cta-lstchain#1312]
https://github.com/cta-observatory/cta-lstchain/blob/cfd9a7f9d8535c75886e2b2c508fc4c10192f53b/lstchain/image/modifier.py#L526-L531

For the spe file, I would always use the get resource path here getting the path from importlib resources instead of trying first to search for the path in the config file.

It happens to me that the path in the lstchain config used for DL1 production was defined as `"lstchain/data/SinglePhE_ResponseInPhE_expo2Gaus.dat"` which does not define the absolute path, hence failing.

**Comment by gabemery** [comment]
@morcuended  I disagree, if some users use a different MC production with a new pulse shape, it should be an option to provide the specific file.
